dropwizard-archetypes/README.md

```diff
@@ -1,8 +1,10 @@
 # Dropwizard Archetypes
 
-How to create project using dropwizard archetype
+How to create project using dropwizard archetype (interactive mode)
 ---
 
 ```
-mvn archetype:generate -DarchetypeGroupId=io.dropwizard.archetypes -DarchetypeArtifactId=java-simple -DarchetypeVersion=0.9.1
+mvn archetype:generate -DarchetypeGroupId=io.dropwizard.archetypes -DarchetypeArtifactId=java-simple -DarchetypeVersion=[REPLACE ME WITH A VALID DROPWIZARD VERSION]
 ```
+
+(when asked for $name during project creation via maven, make sure to use a camel case word such as HelloWorld as it is used to generate Configuration and Application classess such as HelloWorldConfiguration.java and HelloWorldApplication.java. Furthermore, do not include any blank space for the same reason!)
```

Comment:
I think, we could wrap `$name`, `HelloWorld`, and `HelloWorldConfiguration.java` in backticks to highlight them.
Apart from that, this is a very useful tip.